__tests__/ProfileBanner-test.js

```diff
@@ -5,16 +5,47 @@ import ProfileBanner from '@screens/app/Components/ProfileBanner/ProfileBanner';
 import renderer from 'react-test-renderer';
 
 test('Profile Banner Snapshot Test', () => {
-  const tree = renderer
+  const treeMatchingUsername = renderer
     .create(
       <ProfileBanner
         username={''}
         posts_count={0}
         follower_count={0}
         following_count={0}
         profile_image={''}
+        your_id={''}
       />,
     )
     .toJSON();
-  expect(tree).toMatchSnapshot();
+  expect(treeMatchingUsername).toMatchSnapshot();
+
+  const treeNotMatchingUsernameTrueFollowing = renderer
+    .create(
+      <ProfileBanner
+        username={'a'}
+        posts_count={0}
+        follower_count={0}
+        following_count={0}
+        profile_image={''}
+        isFollowing={true}
+        your_id={'b'}
+      />,
+    )
+    .toJSON();
```

Comment:
Consider automating ProfileBanner creation with the use of functions for object/component creation